ui/helpers/utils/util.js

```diff
@@ -631,3 +635,58 @@ export const getNetworkNameFromProviderType = (providerName) => {
 export const isAbleToExportAccount = (keyringType = '') => {
   return !keyringType.includes('Hardware') && !keyringType.includes('Snap');
 };
+
+/**
+ * Converts hex string to decimal value.
+ *
+ * @param {*} hex - string hex to convert to decimal
+ * @returns string decimal
+ */
+
+export const hexToDecimal = (hex) => parseInt(hex, 16);
+
+/**
+ * Converts decimal string to hex string
+ *
+ * @param {*} dec - string to convert to hex
+ * @returns string hex
+ */
+export const decimalToHex = (dec) => {
+  const decimalNumber = parseInt(dec, 10); // 10 is the base for decimal
+  return `0x${decimalNumber.toString(16)}`;
+};
+
+/**
+ * Checks if a tokenId in Hex or decimal format already exists in an object.
+ *
+ * @param {*} address - collection address.
+ * @param {*} tokenId - tokenId to search for
+ * @param {*} obj - object to look into
+ * @returns {boolean} `false` if tokenId does not already exist.
+ */
+export const checkTokenIdExists = (address, tokenId, obj) => {
+  // check if input tokenId is hexadecimal
+  // If it is convert to decimal and compare with existing tokens
+  // if it is decimal convert to hexadecimal and compare
```

Comment:
I'm wondering why it only checks the opposite format.  Shouldn't it always check for duplicates in *both* decimal and hex formats?